resources/views/directory/table.blade.php

```diff
@@ -0,0 +1,88 @@
+@extends('committee.members')
+@section('panel-heading')
+    <div class="panel-heading">Members Directory</div>
+@endsection
+
+@section('tablescript')
+    <script>
+        $(document).ready(function() {
+            var dataObj = {!! $membersJson !!};
+            $('#hot').handsontable({
+                data: dataObj, // see the Data tab
+                columns: [
+                    { data: "full_name" },
+
```

Comment:
This is a **very** nit-picking comment (pardon me!) but maybe there's no need for double spacing here? Empty lines made sense in the other file to separate out related columns, but here it decreases the amount of useful code that one can read in one screen (having a minor effect on readability).